composer.json

```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": ">=5.4",
-    "moxie-lean/wp-endpoint": "1.*.*",
+    "moxie-lean/wp-endpoint": "2.*.*",
```

Comment:
In this way you can control what types of updates you want to introduce to a project major, minor or patches only.

And I think the consistency pattern should be using `^` instead of `*`